src/app/ConfirmPage/index.tsx

```diff
@@ -43,6 +46,7 @@ const ConfirmPage = memo(() => {
         }
       >
         <ConfirmDAppForm />
+        <ToasterProvider />
```

Comment:
Please consider adding `ToasterProvider` into `src/app/App.tsx`: then you will not need to import it here. Moreover, resolving merge conflicts in #1247 will be easier